xls/dslx/bytecode/bytecode_interpreter.cc

```diff
@@ -149,7 +150,9 @@ constexpr int64_t kChannelTraceIndentation = 2;
 
 BytecodeInterpreter::BytecodeInterpreter(
     ImportData* import_data, const BytecodeInterpreterOptions& options)
-    : import_data_(import_data), options_(options) {}
+    : import_data_(import_data),
```

Comment:
Since we're now dereferencing the line below, I suggest to make this `import_data_(ABSL_DIE_IF_NULL(import_data))`

 ... which then will find two places in `bytecode_interpreter_test.cc` that hit this issue.

(Don't worry for now, will do that while importing)

Reply:
Thanks Henner!

(I guess it's just not exercised? I ran on my machine xls/... with `--config=asan` and it was clean, I imagined it'd show if a null deref happened.)

Reply:
Look for `/*import_data=*/nullptr` in `xls/dslx/bytecode/bytecode_interpreter_test.cc`. It might work by accident as nothing is called on it.

Reply:
Ok let me know if you need me to revise, sounded like you were planning to fix it on import but LMK if I misinterpreted.

Reply:
Yep, already fixed that in my local import copy.
Just in the process to get everything else outside xls lined up. To be submitted soon.